_run/common-kind.mk

```diff
@@ -40,7 +42,7 @@ kind-k8s-ip:
 
 .PHONY: kind-configure-image
 kind-configure-image:
-	echo "- op: replace\n  path: /spec/template/spec/containers/0/image\n  value: ${DOCKER_IMAGE}" > ./kustomize/akash-node/docker-image.yaml && \
+	echo "- op: replace\n  path: /spec/template/spec/containers/0/image\n  value: $(DOCKER_IMAGE)" > ./kustomize/akash-node/docker-image.yaml && \
```

Comment:
Can we just have the files there with this default so that this command doesn't need to be run?

Reply:
💯